service/controller/resource/monitoring/pvresizingresource/resource.go

```diff
@@ -0,0 +1,37 @@
+package pvresizingresource
```

Comment:
Are we resizing the PV or the PVC 😅 I know I gave you that naming idea, my apologies